processing/app/main.py

```diff
@@ -10,3 +14,50 @@ def read_root():
 @app.post("/items/")
 def create_item(item: Item):
     return {"item_received": item}
+
+
+def construct_prompt(request:ProcessSurveyRequest) -> str:
+    answer_tags = [f"<answer>{answer}</answer>" for answer in request.answers]
+    answers_xml = "\n".join(answer_tags)
+
+    prompt = f"""
+<input>
+You will be provided with survey responses in `answers` tag, one user answer is in `answer` tag. Question for survey: {request.question}
+</input>
+
+<instruction>
+Rewiew and analize responses from `answers` with goal in mind to find and extract informations that will be relevant for question. Treat similar points like one. Prefix each point accounted in final response with: \"frequent\", \"moderate\", \"occasional\" depending how often it was mentioned.
+</instruction>
+
+<answers>
+{answers_xml}
+</answers>
+    """
+
+    print(prompt)
+
+    return prompt
+
+
+async def ask_ollama_process(request: ProcessSurveyRequest,client: AsyncClient) -> str:
+
```

Comment:
Nit: this empty line shouldn't be here. I highly recommend installing a formatter like ruff and setup code actions to format on save/focus change.